app/src/main/java/org/mifos/selfserviceapp/ui/fragments/LoanAccountsListFragment.java

```diff
@@ -0,0 +1,146 @@
+package org.mifos.selfserviceapp.ui.fragments;
+
+import android.app.ProgressDialog;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.data.accounts.LoanAccount;
+import org.mifos.selfserviceapp.presenters.LoanAccountsListPresenter;
+import org.mifos.selfserviceapp.ui.activities.BaseActivity;
+import org.mifos.selfserviceapp.ui.adapters.LoanAccountsListAdapter;
+import org.mifos.selfserviceapp.ui.views.LoanAccountsListView;
+import org.mifos.selfserviceapp.utils.Constants;
+import org.mifos.selfserviceapp.utils.RecyclerItemClickListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * @author Vishwajeet
+ * @since 30/07/16.
+ */
+
+public class LoanAccountsListFragment extends Fragment implements RecyclerItemClickListener.OnItemClickListener, LoanAccountsListView {
+
+    @Inject
+    LoanAccountsListPresenter mLoanAccountsListPresenter;
```

Comment:
both loan and savings list fragment look very similar. Do you see them being one class? May want to keep an eye on this in future, if they keep behaving similar, would be useful to keep them as the same fragment.